blog/wg_announcement/index.qmd

```diff
@@ -0,0 +1,23 @@
+---
+title: "New ASA BIOP working group on Software Engineering"
+author: "SWE WG"
+subtitle: Fostering open source code collaboration within Biostatistics
+description: We are officially announcing the new ASA BIOP scientific working group on Software Engineering (SWE WG).
+date: "2022-10-24"
+categories: [news]
+image: "sticker-1200.png"
+---
+
+With the shift of biostatistics work from proprietary software (e.g. SAS) to R, it is now even more important to write reliable and reproducible code, and to be selective with the R packages we use. This topic is equally relevant for biostatistics work as it is for programming and other roles. A recent [panel discussion](https://www.linkedin.com/pulse/research-software-engineering-clinical-biostatistics-saban%C3%A9s-bov%C3%A9/) on this topic highlighted this evolving need across companies and academia.
+
+To foster collaboration within biostatistics across organizational silos we proposed a new scientific working group under the umbrella of the [American Statistical Association (ASA) Biopharmaceutical section (BIOP)](https://community.amstat.org/biop/home) which was quickly approved in August this year: The Software Engineering working group (SWE WG). Now we have launched our [landing page](https://community.amstat.org/biop/workinggroups/swe-wg) and [website](https://rconsortium.github.io/asa-biop-swe-wg/) (hosted by the [R Consortium](https://www.r-consortium.org/about) GitHub organization) and want to share this news broadly to raise awareness and attract contributors. We have more than 10 companies and institutions involved in the SWE WG and welcome new members.
+
+The primary objective of the SWE WG is to engineer R packages that implement important methods that are often utilized in biostatistics but still missing from our R toolbox. The first product is the recently-published R package implementing mixed models for repeated measures (MMRM): [mmrm](https://cran.r-project.org/package=mmrm). In particular, a critical advantage of this package over existing implementations is that it is at least 3-5 times faster and converges more reliably. It also provides a comprehensive set of features: users can specify a variety of covariance matrices, weight observations, fit models with restricted or standard maximum likelihood inference, perform hypothesis testing with Satterthwaite adjusted degrees of freedom, and extract the least square means estimates by using the [emmeans package](https://github.com/rvlenth/emmeans). We aim to establish the R package [mmrm](https://cran.r-project.org/package=mmrm) as a new standard within biostatistics and will communicate it broadly.
```

Comment:
Make it clear that collaboration on mmrm is open to anyone - it is just initiated by the ASA WG?

Reply:
added a sentence